src/Native/Bootstrap/platform.unix.cpp

```diff
@@ -70,4 +70,24 @@ extern "C"
     {
         throw "RhCompatibleReentrantWaitAny";
     }
+
+    void EnumDynamicTimeZoneInformation()
```

Comment:
Is this only on Unix? `--ilcpath` should supply them with the ILC and ILC SDK path and hence the live CoreLib inside `sdk/System.Private.CoreLib.dll` in `stage1`. This is very surprising. On Windows, I clearly remember editing S.P.CoreLib during the thread statics PR and it failed/fixed with my changes.